test/com/google/enterprise/adaptor/DocIdSenderTest.java

```diff
@@ -491,6 +564,48 @@ public void testAclItemToString() {
     assertEquals(golden, "" + new DocIdSender.AclItem(id, acl));
   }
 
+  private static Map<GroupPrincipal, Collection<Principal>> sampleGroups() {
```

Comment:
Please move this back to the top. I dislike buried test data, it makes the tests read like too much magic. I assume you created this method because having the test data replicated in each test was too messy? I'm not sure that I entirely agree, but at least let's put this back first and see how it reads.